src/main/java/ch/njol/skript/effects/EffBrewingConsume.java

```diff
@@ -0,0 +1,58 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.inventory.BrewingStandFuelEvent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Consume Brewing Fuel")
+@Description({
+	"Make the brewing fuel event consume the fuel",
+	"By making it not consume, it will keep the fuel item and still add fuel to the brewing stand."
```

Comment:
```suggestion
	"Makes the brewing stand in the brewing fuel event consume its fuel.",
	"By making it not consume the fuel, it will keep the fuel item and still add to the fuel level of the brewing stand."
```